build.sbt

```diff
@@ -4,31 +4,32 @@ import interplay.ScalaVersions
 
 import scalariform.formatter.preferences._
 
-lazy val commonSettings = mimaDefaultSettings ++ SbtScalariform.scalariformSettings ++ Seq(
+lazy val commonSettings = mimaDefaultSettings ++ Seq(
   // scalaVersion needs to be kept in sync with travis-ci
   scalaVersion := ScalaVersions.scala212,
   crossScalaVersions := Seq(ScalaVersions.scala211, ScalaVersions.scala212),
+  scalariformAutoformat := true,
   ScalariformKeys.preferences := ScalariformKeys.preferences.value
     .setPreference(SpacesAroundMultiImports, true)
     .setPreference(SpaceInsideParentheses, false)
     .setPreference(DanglingCloseParenthesis, Preserve)
     .setPreference(PreserveSpaceBeforeArguments, true)
-    .setPreference(DoubleIndentClassDeclaration, true)
+    .setPreference(DoubleIndentConstructorArguments, true)
 )
 
 // needs to be kept in sync with travis-ci
-val PlayVersion = playVersion(sys.env.getOrElse("PLAY_VERSION", "2.6.11"))
+val PlayVersion = playVersion(sys.env.getOrElse("PLAY_VERSION", "2.6.18"))
 
 // Version used to check binary compatibility
-val mimaPreviousArtifactsVersion = "6.0.0"
+val mimaPreviousArtifactsVersion = "6.0.1"
```

Comment:
This should stay at `6.0.0` I think.

Reply:
Thanks for taking a look at this PR!

I've rolled back the version to `6.0.0` in the latest commits.